alliance/apps/backlog/views/check_backlogs_view.py

```diff
@@ -33,26 +34,22 @@ def post(self, request):
                 #  existing into database.
                 for ui_backlog in ui_backlogs:
                     backlog_id = ui_backlog.get('id')
-                    try:
-                        backlog = Backlog.objects.get(id=backlog_id)
-                    except Backlog.DoesNotExist:
-                        # Maybe this backlog has been deleted by an
-                        #  admin user.
-                        result['outdated'] = True
-                    else:
-                        # Database datetime has greater precision than
-                        #  the one brought from ui so we add a timedelta,
-                        #  otherwise the page that executed the update
-                        #  would show the message of requested refresh.
-                        ui_last_update = parse_datetime(
+                    backlog = Backlog.objects.get_object_or_none(id=backlog_id)
+
+                    result['outdated'] = backlog == None
```

Comment:
We should do comparison to `None` with the `is` keyword instead of `==`. `result['outdated'] = backlog is None`

http://stackoverflow.com/questions/14247373/python-none-comparison-should-i-use-is-or